### ISSUE TITLE
Disable globally "scan all"

### ISSUE THREAD

**Issue by KryLex0** [goharbor/harbor#20751]
Hello,

I would like to suggest an improvement for the project.

Currently, the "scan all" functionality is enabled and cannot be disabled. Even an administrator can accidentally perform this action, which could pose a significant risk in a production environment.

I suggest allowing the "scan all" functionality to be globally disabled from the web UI and the API, with only superadmin rights permitting this action.

This request follows an issue that occurred a few days ago.

Version used: 2.8.9

**Comment by wy65701436** [comment]
Can you explain the significant risk a bit more? You can stop the scan job at any time after it starts. Isn't that useful?

**Comment by KryLex0** [comment]
Yes, the scan can be stopped, but we encountered an issue where the scan job had already started, causing images to be unpullable due to the ongoing scan process. This affected many projects in the production environment. 

Even after stopping the scan, the images couldn't be pulled until we rescanned them individually. We had to do this because the scan all job looped over a large number of images. 
While it doesn't pose a risk in terms of vulnerabilities, it can impact projects that need to pull images from Harbor. Having the possibility to disable this action could prevent this kind of problem from happening again.
